contracts/utility-token/UtilityToken.sol

```diff
@@ -51,6 +49,50 @@ contract UtilityToken is ERC20Token {
 
     /* External Functions */
 
+    /**
+     * @notice Sets up the symbol, name, decimals, totalSupply, value token
+     *         and the consensusCogateway address
+     *
+     * @param _symbol Symbol of token.
+     * @param _name Name of token.
+     * @param _decimals Decimal of token.
+     * @param _totalTokenSupply Total token supply.
+     * @param _consensusCogateway ConsensusCogateway contract address.
+     * @param _valueToken Address of value token contract.
+     */
+    function setup(
+        string calldata _symbol,
+        string calldata _name,
+        uint8 _decimals,
+        uint256 _totalTokenSupply,
+        address _consensusCogateway,
+        address _valueToken
+    )
+        external
+    {
+        require(
+            consensusCogateway == address(0),
+            "ConsensusCogateway address is already set."
```

Comment:
I am assuming we do this check to make sure contract is initialized/setup only once. Hence, I would propose to update the require message to `Contract has been already setup`.

Reply:
Done